R/deploy-site.R

```diff
@@ -116,7 +116,28 @@ deploy_to_branch <- function(pkg = ".",
   dest_dir <- fs::dir_create(fs::file_temp())
   on.exit(fs::dir_delete(dest_dir))
 
-  git("fetch", "origin", branch)
+  tryCatch(
```

Comment:
There's no easy way to find if the branch exists without throwing an error?

Reply:
I could use `git ls-remote` to check if it exists, but we would still have to fetch the remote anyway...